submissions/description/mobile_security_misconfiguration/ssl_certificate_pinning/absent/recommendations.md

```diff
@@ -1,6 +1,6 @@
 # Recommendation(s)
 
-It is recommended to implement SSL certificate pinning for the application.
+It is recommended to implement Sockets Layer (SSL) certificate pinning for the application.
```

Comment:
Missing "Secure" here

Reply:
Should we be migrating each instance of SSL to TLS, while it's somewhat universal, SSL as a term is deprecated and has been for a long time.